storage/ndb/src/kernel/blocks/ndbcntr/NdbcntrMain.cpp

```diff
@@ -6187,6 +6187,8 @@ void Ndbcntr::open_secretsfile(Signal *signal,
   ekm.length = globalData.filesystemPasswordLength;
   memcpy(&ekm.data, globalData.filesystemPassword, globalData.filesystemPasswordLength);
   ekm.data[globalData.filesystemPasswordLength] = 0;
+  ekm.data[globalData.filesystemPasswordLength + 1] = 0;
+  ekm.data[globalData.filesystemPasswordLength + 2] = 0;
```

Comment:
Why is this helping, some comment on why this fix is required would be helpful.

Reply:
We send a number of `Uint32`s in the signal, but we only initialize `length+1` bytes, so some bytes can remain uninitialized. However, it seems that the receiver only uses `length` bytes so perhaps this isn't needed after all.